bokeh/models/transforms.py

```diff
@@ -0,0 +1,107 @@
+'''
+
+'''
+from __future__ import absolute_import
+
+from ..core.enums import StepMode, JitterRandomDistribution
+from ..core.properties import abstract
+from ..core.properties import Either, Enum, Float, Instance, Seq, String, Tuple
+from ..model import Model
+from .sources import ColumnDataSource
+
+@abstract
+class Transform(Model):
+    ''' Base class for ``Transform`` models that represent a computation
+    to be carried out on the client-side.
+
+    JavaScript implementations should implement the following methods:
+
+    .. code-block: coffeescript
+
+        compute: (x) ->
+            # compute the transform of a single value
+
+        v_compute: (xs) ->
+            # compute the transform of an array of values
+
+    '''
+    pass
+
+
+class Jitter(Transform):
+    ''' Apply either a uniform or normally sampled random jitter to data.
+
+    '''
+
+
+    mean = Float(default=0, help="""
+    The central value for the random sample
```

Comment:
I don't think it makes sense for this to be exposed? I've never seen any use of jitter that was not symmetric about the data points.